content.en/polkadot/dht/2024-23-report.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Week 2024-23
+plotly: true
+slug: 2024-23
+weight: 1045636
+---
+
+# Polkadot Report Cal. Week 23 - 2024
+
+## General Information
+
+The following results show measurement data that were collected in calendar week 23 in 2024 from `2024-06-03` to `2024-06-10`.
+
+- Number of crawls `336`
+- Number of visits `7,693,028`
+  > Visiting a peer means dialing or connecting to it. Every time the crawler or monitoring process tries to dial or connect to a peer we consider this as _visiting_ it. Regardless of errors that may occur.
+- Number of unique peer IDs visited `11,027`
+- Number of unique peer IDs discovered in the DHT `10,655`
+- Number of unique IP addresses found `16,947`
+
+Timestamps are in UTC if not mentioned otherwise.
+
```

Comment:
As they are multiple user agents, we should add an _Agents_ section, which can be identical to the [one from the Ethereum reports](https://probelab.io/ethereum/discv5/2024-19/#agents).